src/utils.jl

```diff
@@ -152,6 +152,8 @@ end
 function getsource(source, use_mmap)
     if source isa Vector{UInt8}
         return source
+    elseif source isa Cmd
+        return Base.read(source)
```

Comment:
Hmmmm, this makes me a bit worried, if you happened to do something like `CSV.read(`gunzip really_large_csv.csv.gz`)`, we might run out of memory, but I'm also not sure what other option we would have. I guess this is probably fine.